2019-05-Berlin/agenda.md

```diff
@@ -0,0 +1,86 @@
+
+Our time will be best served with some monotrack presentations and discussions in the first day, and then breakout rooms in the second day. __We can change everything__, so if you think a better structure will suit us better, let us know in #135!
+
+It would be fantastic if each one of the breakouts is facilitated by a person who will be responsible to "run" the breakout and do a quick presentation at the end. This person would also set outcomes from the breakout to  optimally leverage our time together.
+
+✨ We're excited for Node.js' 5th round of collaboration IRL! Even better, we're excited that folks from the JSF will be joining us for the first time! ✨
+
+> __Location TBA__,
+(Street Address)
+Berlin, Germany
+
+
+## Collaborator Summit Day One: 30th May, 2019
+
+(Time window)
+
+### Agenda
+
+(Types of rooms available):
+
+* (details of main room)
+* (details of more rooms)
+
+_(Expected participation number)_.
+
+
+| Time          | Place        | Topic                                         | Speakers/Champions                     | Issue                                      | Slides                                    |
+|---------------|--------------|-----------------------------------------------|----------------------------------------|--------------------------------------------|-------------------------------------------|
+| | Main         | Intro, Sponsor Talk, Code of Conduct          | |                                            | |
+
+
+_To speak/present a topic/do a sesssion, make a pull request to update this document with a title, abstract, and a link to an issue with more detail in this repo, similar to https://github.com/nodejs/summit/issues/86_.
+
+
+## Collaborator Summit Day Two: 31st May, 2019
+
+(time window)
+
+### Agenda
+
+(Types of rooms available):
+
+* (details of main room)
+* (details of more rooms)
+
+_(Expected participation number)_.
+
+
+| Time          | Place        | Topic                                                               | Speakers/Champions            | Issue                                      | Slides                                    |
+|---------------|--------------|---------------------------------------------------------------------|-------------------------------|--------------------------------------------|-------------------------------------------|
+| | Main | Intro, Code of Conduct                                              | |                                            | |
+
+
+_To speak/present a topic/do a sesssion, make a pull request to update this document with a topic, contact @github-handle. Include a link to an issue in this repo with more detail, similar to https://github.com/nodejs/summit/issues/86_ that contains at least an abstract, a link to the planning issue in the working group repo, and any extra requests (minimum space during conf, av/sound, etc.).
+
+
+## Organizers
+
+- name (@github-handle)
```

Comment:
```suggestion
- Matteo Collina (@mcollina)
```

(But there are also a lot of other people, including yourself ;))

Reply:
Right on, I'll add myself ;) and everyone who's expressed willingness in the issue